index.html

```diff
@@ -1,232 +1,283 @@
+    />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <script
+      src="https://kit.fontawesome.com/493f30a6ae.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
 
-<head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <meta name="author" content="Felipe Oyarzo" />
-  <title>Felipe Oyarzo's Portfolio</title>
-  <link rel="preconnect" href="https://fonts.googleapis.com" />
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-  <link
-    href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;700&display=swap"
-    rel="stylesheet" />
-  <link rel="stylesheet" href="./assets/css/style.css" />
-</head>
-
-<body>
-  <header>
-    <nav class="nav-bar">
-      <div id="menu-wrapper">
-        <a href="#hello">Hello</a>
-        <a href="#portfolio">Portfolio</a>
-        <a href="#about">About</a>
-        <a href="#contact-section">Contact</a>
-      </div>
-      <a href="#"><img src="./assets/img/ic_menu.png" alt="Hamburguer menu icon" id="hamburguer" /></a>
-    </nav>
-  </header>
-  <main>
-    <section class="headline" id="hello">
-      <div class="text-wrapper">
-        <h1>Felipe<br />Oyarzo</h1>
-        <p>
-          Hello! I am a software developer! I can help you build a product,
-          feature or website. Take a look of my works. If you like what you
-          see and have a project you need coded, don't hesitate and contact
-          me.
-        </p>
-        <button class="button" type="button" id="btnHeadline">
-          Start collaboration
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <button type="button" class="menuIcon">
+          <i class="fa-solid fa-bars fa-2xl"></i>
         </button>
-      </div>
-    </section>
-    <section class="projects" id="portfolio">
-      <h2>Projects</h2>
-      <div class="grid-container">
-        <div class="grid-layout layout1"></div>
-        <div class="grid-layout layout2"></div>
-        <div class="grid-layout layout3"></div>
-        <div class="grid-layout layout4"></div>
-        <div class="grid-layout layout5"></div>
-        <div class="grid-layout layout6"></div>
-        <div class="card card1">
-          <div class="card-image image1"></div>
-          <div class="card-body">
-            <h3 class="project-title">Project name goes here</h3>
-            <ul class="card-list">
-              <li class="card-list-item">HTML/CSS</li>
-              <li class="card-list-item">Ruby on Rails</li>
-              <li class="card-list-item">JavaScript</li>
-            </ul>
-            <button type="button" class="card-btn">
-              See this project &rarr;
-            </button>
-          </div>
+        <button type="button" class="closeIcon button hidden">
+          <i class="fa-solid fa-xmark fa-2xl"></i>
+        </button>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.